docs/glossary.md

```diff
@@ -2,76 +2,78 @@
 
-## Impact model
+Source: Adapted from ([UNDRR](https://www.undrr.org/terminology/disaster))
 
-An impact model (also damage model; loss model) is used to combine Hazard, Exposure and Vulnerability components to calculate the amount of value that is expected to be impacted (damaged or lost).
 
-Source: [GFDRR, Solving the Puzzle, 2016](https://www.gfdrr.org/sites/default/files/solving-the-puzzle-report.pdf)
+## Loss
+Losses are a measure of this impact in the form of damage or destruction caused by a disaster (observed or modelled). This measure may be expressed as monetary (e.g. cost of reconstruction) or non-monetary (e.g. hectares of crop loss, loss of life). Losses can be direct or indirect. 
 
-```{card} Impacts and losses: USD costs of replacement
-![Loss](img/sample_lss.jpg)
-```
+The total economic impact consists of a monetary estimate of both direct disaster loss and indirect disaster loss. _Direct disaster losses_ refer to directly quantifiable losses such as the number of people killed and the damage to buildings, infrastructure and natural resources. _Indirect disaster losses_ refer to wider and longer-term impacts arising from the direct disruptions to the flow of goods and services as a result of a disaster. This includes declines in output or revenue, increases in food prices, and other longer-term social and economic effects on education, health, productivity, the well-being of people or the macroeconomy. 
 
-## Vulnerability
+Source: Adapted from ([UNDRR](https://www.preventionweb.net/understanding-disaster-risk/key-concepts/direct-indirect-losses))
+and [UNDRR Terminology (Economic Loss)](https://www.undrr.org/terminology/economic-loss)
 
-Vulnerability are the conditions determined by physical, social, economic and environmental factors or processes which increase the susceptibility of an individual, a community, assets or systems to the impacts of hazards. The inverse of vulnerability is defined coping capacity, which is the combination of all the strengths, attributes and resources available within an organization, community or society to manage and reduce disaster risks and strengthen resilience.
 
-Source: [UNDRR Terminology (Vulnerability)](https://www.undrr.org/terminology/vulnerability)
+## People affected
+People who have been exposed and impacted, either directly or indirectly, by a hazardous event. Directly affected are those who have suffered injury, illness or other health effects; who were evacuated, displaced, relocated or have suffered direct damage to their livelihoods, economic, physical, social, cultural and environmental assets. Indirectly affected are people who have suffered consequences, other than or in addition to direct effects, over time, due to disruption or changes in the economy, critical infrastructure, basic services, commerce or work, or social, health and psychological consequences.
```

Comment:
```suggestion
People who have been exposed and impacted, either directly or indirectly, by a hazardous event. 

Directly affected are those who:
* have suffered injury, illness or other health effects,
* were evacuated, displaced, or relocated,
* have suffered direct damage to their livelihoods, economic, physical, social, cultural and environmental assets. 

Indirectly affected are people who have suffered consequences, other than or in addition to direct effects, over time, due to:
* disruption or changes in the economy, critical infrastructure, basic services, commerce or work,
* social, health or psychological consequences.
```

Using bullet pointed lists just makes it a bit easier to parse these lists of lists.